6213 ЧегуровНС Лаб.4

Chegur opened the pull request.

Chegur committed:
Add files via upload

Chegur committed:
Update 1lab.html

Chegur committed:
Create style.css

Chegur committed:
Загрузил style.css

Chegur committed:
Update style.css

Убрал блок image и сделал image-container__background и image-container__foreground модификаторами: image_container--background и image-container--foreground

Chegur committed:
Update 1lab.html

убрал image
image-container__background -> image-container--background
image-container__foreground -> image-container--foreground

Chegur committed:
Create style2.css

Chegur committed:
added 2lab.html

Chegur committed:
Перелопатил 2lab.html

Изменил логику классов во 2-ом задании теперь child - элементы parent с конкретным номером (family__parent-1__child)
Для 3-его задания добавил <div> в которое оно оборачивается + добавил классы для каждого элемента

Chegur committed:
Перелопатил style2.css

Все как в HTML только добавил лого в конце списка

Chegur committed:
изменил названия классов style2.css

Chegur committed:
Тут тоже изменил названия классов

Chegur committed:
Create 3lab.html

А все, просто создал папку и файл

Chegur committed:
Добавил файл фотографии профиля и css

Chegur committed:
Исправил 3lab.html

profile__hobby на hobbies__hobby
profile__skill-item на skill-list__skill-item

Chegur committed:
Исправил style3.css

Chegur committed:
Выгрузил лабу 4

Простенький калькулятор, фантазия у меня есть, но дедлайны дороже)

AvtoBBus commented on 4lab/script.js:
```diff
@@ -0,0 +1,45 @@
+document.getElementById('calculate').addEventListener('click', () => {
+  const num1Input = document.getElementById('num1');
+  const num2Input = document.getElementById('num2');
+  const operation = document.getElementById('operation').value;
+  const resultOutput = document.getElementById('result');
```
А если какой-то элемент не был найден?

AvtoBBus commented on lab3/3lab.html:
Как-то не наблюдается ссылок на остальные 3 лабы

AvtoBBus reviewed (changes requested):
Странный дизайн в плане расположения блоков, сделайте хотя бы вот так в столбец:

<img width="379" height="413" alt="Image" src="https://github.com/user-attachments/assets/3a1cdb72-2ee0-4487-906e-674599456f06" />

Chegur committed:
Delete 4lab/lab1.html

Chegur committed:
Update lab4.html

Доработал БЭМ, и обернул ввод и операцию в дополнительный класс для более красивого вида

Chegur committed:
Update script.js

Добавил проверку на существование элементов

Chegur committed:
Update style.css

Добавил контейнер для операции и переменных, исправил БЭМ, немного поменял параметры для лучшего отображения

Chegur committed:
Обновил 3lab.html для 4 лабы

Chegur committed:
Обновил style3.css для Лабы 4

Добавил навигацию по лабам

Chegur committed:
Исправил табуляцию style3.css

Chegur committed:
Добавил файлы всех лабораторных для навигации

Добавил файлы всех лабораторных для навигации

AvtoBBus commented:
Принято
